### DISCUSSION TITLE
How to Remove All Items from Select Box

### DISCUSSION THREAD

**Discussion by Ravikumar15081987** [heroui-inc/heroui#2166] [Q&A]
Hi, 

I want to reset the select box, remove all the SelectItem from select box before insert new items to Select Box, Please give me solution for this.

My Select Box 

const [packHeadData, setPackHeadData] = useState([]);

<Select
      label="Package Heading" placeholder="Select Package Heading" className="max-w-xs" variant={"bordered"}
      radius="none" labelPlacement="outside" selectionMode='single'
>
      {packHeadData.map((packHead: PackageHeading) => (
            <SelectItem key={packHead.id} value={packHead.id}>
                  {packHead.heading}
            </SelectItem>
      ))}
</Select>

Note: setPackHeadData([])   --- Not Working

**Comment by boompttn** [comment]
You need to make it controlled Select

https://nextui.org/docs/components/select#controlled
